README_dev.md

```diff
@@ -38,14 +46,16 @@ Mage.ai server uses at least Python 3.6 (as per `setup.py`), but the development
 As such, make sure you have Python >=3.8. Verify this with:
 
 ```bash
-python --version
+git clone https://github.com/mage-ai/mage-ai mage-ai
+cd mage-ai
```

Comment:
Matt, my man, why cloning and cd-ing into the directory help verify the correct Python version?